back/engines/commercial/user_custom_fields/spec/acceptance/ref_distributions_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Representativeness reference distributions' do
```

Comment:
I know that it was not done in the past, but in `citizenlab/back/engines/commercial/content_builder/spec/acceptance/layouts_spec.rb` we have added examples to describe what happens when authorization fails. Without those examples, the API user does not know which http status will be returned. This is not a blocker for this PR. I just wanted to share it. Probably at some point we need a discussion about it at the CDM.

Reply:
Good catch! Actually, this is something I usually do, a bit differently though: `engines/commercial/insights/spec/acceptance/views_spec.rb`. I will add them.